tests/transaction_service_test.go

```diff
@@ -0,0 +1,78 @@
+package tests
+
+import (
+	"context"
+	"strings"
+	"testing"
+
+	"github.com/flow-hydraulics/flow-wallet-api/templates"
+	"github.com/flow-hydraulics/flow-wallet-api/tests/internal/test"
+	"github.com/flow-hydraulics/flow-wallet-api/transactions"
+	"github.com/onflow/flow-go-sdk"
+)
+
+func Test_Transaction_Sign_By_Admin(t *testing.T) {
+	cfg := test.LoadConfig(t, testConfigPath)
+	txSvc := test.GetServices(t, cfg).GetTransactions()
```

Comment:
Encapsulating support functionality for service composition & building make test cases a bit clearer this way.